bittensor/_cli/cli_impl.py

```diff
@@ -548,15 +548,26 @@ def overview(self):
 
             for index, column in zip(range(len(table.columns)), table.columns):
                 # Fuzzy match the column name. Default to the first column.
-                if fuzz.ratio(sort_by.lower(), column.header.lower().replace('[overline white]', '')) > 80:
+                column_name = column.header.lower().replace('[overline white]', '')
+                match_ratio = fuzz.ratio(sort_by.lower(), column_name)
+                if  match_ratio > 70:
```

Comment:
can you add to constants.py?

Reply:
changed to use the maximum match rather than a threshold